src/Breakpoint.php

```diff
@@ -142,66 +177,14 @@ private function getFormatQuality(string $format = null): int|null
         return null;
     }
 
-    private function getWidths(): Collection
-    {
-        $params = $this->getParams();
-
-        return app(WidthCalculator::class)
-            ->calculateWidthsFromAsset($this->asset)
-            /* If a width is specified, consider it a max width */
-            ->when(isset($params['width']) || isset($params['w']), function ($widths) use ($params) {
-                $maxWidth = $params['width'] ?? $params['w'];
-
-                $filtered = $widths->filter(function (int $width) use ($maxWidth) {
-                    return $width <= $maxWidth;
-                });
-
-                /* We want at least one width to be returned */
-                if (! $filtered->count()) {
-                    $filtered = collect($maxWidth);
-                }
-
-                return $filtered;
-            });
-    }
-
-    public function buildImageJob(int $width, ?string $format = null, ?float $ratio = null): GenerateImageJob
-    {
-        $params = $this->getParams($format);
-
-        $params['width'] = $width;
-
-        if (! is_null($ratio)) {
-            $params['height'] = $width / $ratio;
-        }
-
-        return app(GenerateImageJob::class, ['asset' => $this->asset, 'params' => $params]);
-    }
-
-    public function dispatchImageJobs(): void
-    {
-        $this->getWidths()
-            ->map(function (int $width) {
-                dispatch($this->buildImageJob($width, null, $this->ratio));
-                if (config('statamic.responsive-images.webp', true)) {
-                    dispatch($this->buildImageJob($width, 'webp', $this->ratio));
-                }
-
-                if (config('statamic.responsive-images.avif', false)) {
-                    dispatch($this->buildImageJob($width, 'avif', $this->ratio));
-                }
-            });
-    }
-
     public function toArray(): array
     {
         return [
             'asset' => $this->asset,
             'label' => $this->label,
-            'value' => $this->value,
-            'media' => $this->getMediaString(),
+            'minWidth' => $this->minWidth,
```

Comment:
`value` in my opinion felt too vague as property name, so I have renamed it to `minWidth`.

Reply:
👍